pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools >= 77.0.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pals-schema"
+version = "25.08"
```

Comment:
I saw that one could also do
```toml
dynamic = ["version"]
```
but I have not checked how that works with Setuptools specifically.

Reply:
Fancy :) Let me know when you find out :)

Reply:
Oh, one last suggestion: Maybe start with a `0.1` version scheme.

We can always go to a `YY.MM` scheme later if we want, but due to the ordering one _cannot go back_ from it easily!

Reply:
Got it, done in #20.